authentication/certificate-types-descriptions.adoc

```diff
@@ -0,0 +1,36 @@
+[id="ocp-certificates"]
+= Certificate types and descriptions
+include::modules/common-attributes.adoc[]
+:context: ocp-certificates
+
+toc::[]
+
+== Certificate validation
+
+{product-title} monitors certificates for proper validity, for the cluster
+certificates it issues and manages. The {product-title} alerting framework has
+rules to help identify when a certificate issue is about to occur. These rules
+consist of the following checks:
+
+* API server client certificate expiration is less than five minutes.
```

Comment:
> These rules consist of the following checks

Only one check is seen currently, seems not complete here.
For `API server client certificate expiration is less than five minutes`, I'm not sure if `less than five minutes` correct, need Dev help.

Reply:
Thank you! I will work with Engineering to confirm.